### ISSUE TITLE
Issue with stringr::str_sub() within mutate()

### ISSUE THREAD

**Issue by polarimetric** [xiaodaigh/disk.frame#276]
I'm experiencing a bizarre issue with disk.frame that I believe is somehow related to quosures. This is the exact code I'm running, with variable names omitted:

`panel_nutrition <- panel_df %>% mutate(Panel_Month = str_sub(purchase_date, 6, 7)) %>% inner_join(products_nutrition, by='...') %>% mutate_at(vars(...)), ~.*quantity) %>% group_by(...) %>% summarize_at(vars(...), ~sum(., na.rm=TRUE)) %>% collect`

The code fails after a seemingly random amount of time. Each time it fails, I get a syntax error from disk.frame:

```
Error in value[[3L]](cond) :
disk.frame has detected a syntax error in 
chunk_fn(as.data.frame(.x), Panel_Month = str_sub(purchase_date, 6, 7)). If you believe your syntax is correct, raise an issue at https://github.com/xiaodaigh/disk.frame with a MWE
In addition: Warning message:
Using `as.character()` on a quosure is deprecated as of rlang 0.3.0.
Please use `as_label()` or `as_name()` instead.
This warning is displayed once per session.
```

Interestingly, it also only happens after running setup_disk.frame() to parallelize disk.frame operations. Here's a MWE:

```
library(disk.frame)
library(tidyverse)
example <- as.disk.frame(data.frame(purchase_date=c("2020-03-20","2020-04-20")))
example %>% mutate(Panel_Month = str_sub(purchase_date, 6, 7)) %>% collect
# works as intended
setup_disk.frame()
example %>% mutate(Panel_Month = str_sub(purchase_date, 6, 7)) %>% collect
# fails with above error
```

This is on `R 3.6.0` with `disk.frame 0.3.4`.

**Comment by polarimetric** [comment]
It also works as intended if I include stringr:: in the command, so maybe it's actually related to namespace exporting to the `future` workers:

`example %>% mutate(Panel_Month = stringr::str_sub(purchase_date, 6, 7)) %>% collect`
`# works as intended`

**Comment by xiaodaigh** [comment]
Thanks for the report. I will look into this ASAP. In general, ::  is a good idea and the code relies on a package to find the right function so eithet it can't find the function or NSE has issues

**Comment by xiaodaigh** [comment]
I will address this in #280. As the changes would be complicated, it might take a while. Just wanted to set the expectation straight.

**Comment by xiaodaigh** [comment]
fixed by #370
